FreeRTOS/Lab-Project-FreeRTOS-FAT
# Bugfix/zynq mount fix2

Description
-----------
Fixing Mounting issues - when mounting with zynq.

Test Steps
-----------

Won't mount unless the mountFailIgnore is set.  Should this be an option/configuration?

Checklist:
----------


Note Unable to add unit tests as this needs to be tested with Zynq.

Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
